src/store/data-slice.js

```diff
@@ -1,102 +1,37 @@
 import { createSlice } from "@reduxjs/toolkit";
+import { RingLoader } from "react-spinners";
 
 const initialState = {
-  data: [],
+  data: { content: [], pageCount: 0 },
```

Comment:
why you don't set page default size? don't you get NaN in first call? Check network

Reply:
I checked, I don't get NAN